femtools/Read_GMSH.F90

```diff
@@ -111,21 +125,56 @@ function read_gmsh_simple( filename, quad_degree, &
          access="stream", form="formatted" )
 
     ! Read in header information, and validate
-    call read_header( fd, lfilename, gmshFormat )
+    call read_header( fd, lfilename, gmshFormat, versionNumber )
+
+    if (versionNumber%major .eq. 4) then
```

Comment:
I know this isn't adhered to in the rest of this module (which is generally a bit messy), but can we stick with ==, /=, < and > rather than .eq., .neq., .lt. and .gt.

Reply:
No problem, I can change these.